### PR TITLE
Css fix

### PR DESCRIPTION
**Pull request by rpinoon**
updated styling for components and added responsiveness for large screens.

### CODE REVIEW

**Comment by normancapule** [comment]
Please add some screenshots next time. The PR is `Css fix` - its hard to visualize with just code.

File: components/styled/bookshelf.ts [hunk 1]
```diff
@@ -5,21 +5,42 @@ const Container = tw.div`
   grid
```
Comment (normancapule) [inline components/styled/bookshelf.ts:5]: Bdw feel free to move these `styles` into its own file. Especially if its bloating the main component

**Review by normancapule — APPROVED** [review]

**Review by mikodagatan — APPROVED** [review]
Looks good to me

File: components/styled/bookshelf.ts (hunk 1 above)
Comment (rpinoon) [reply components/styled/bookshelf.ts:5]: copy that. will do

**Pull request merged**